centos_6_py3/cloudify-wagon-builder.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+echo "Starting..."
+
+CONSTRAINTS_FILE=/packaging/constraints.txt
+REQUIRMENTS_FILE=/packaging/dev-requirements.txt
+
+if test -f /packaging/extra-packaging-instructions.sh
+then
+    source /packaging/extra-packaging-instructions.sh
+fi
+source /env/bin/activate
+echo "manylinux1_compatible = False" > "/env/bin/_manylinux.py"
+if test -f ${CONSTRAINTS_FILE}
+then
+    echo "## $CONSTRAINTS_FILE exist"
+    wagon create -r ${REQUIRMENTS_FILE} -v -f -a '--no-cache-dir -c '${CONSTRAINTS_FILE}'' .
+else
+    echo "## $CONSTRAINTS_FILE doesn't exist"
+    wagon create -r ${REQUIRMENTS_FILE} -v -f .
```

Comment:
@ahmadiesa-abu Generating the wagon name using the new version from the wagon will create a wagon name without the distro `cloudify_openstack_plugin-3.2.12-py36-none-linux_x86_64.wgn` so no way here to distinguish if this is for centos or redhat or ubuntu, specially when it comes to upload the gihub. I think we need to test this for centos + redhat and watch out the generated name again if they are the same then we will end up with two files have the same names which will be issue when publish them to github when generated release 

CC @EarthmanT

Reply:
@mabuaisha I added a rename step after creating the wagon file , that matches what wagon 0.3.2 used to have in the file name so yeah now the naming convention is same as before 

CC @EarthmanT